go.mod

```diff
@@ -2,7 +2,9 @@ module github.com/ipfs/go-ipfs-blockstore
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/gammazero/kmutex v1.0.0
 	github.com/hashicorp/golang-lru v0.5.4
+	github.com/im7mortal/kmutex v1.0.0 // indirect
```

Comment:
After changing to `v1.0.1` and running `go mod tidy`, this should disappear.

Reply:
Done